src/dir/glob.cr

```diff
@@ -1,266 +1,299 @@
 class Dir
+  # Returns an array of all files that match against any of *patterns*.
+  #
+  # The pattern syntax is similar to shell filename globbing, see `File.match?` for details.
+  #
+  # NOTE: Path separator in patterns needs to be always `/`. The returned file names use system-specific path separators.
   def self.[](*patterns) : Array(String)
     glob(patterns)
   end
 
+  # ditto
   def self.[](patterns : Enumerable(String)) : Array(String)
     glob(patterns)
   end
 
-  def self.glob(*patterns) : Array(String)
-    glob(patterns)
+  # Returns an array of all files that match against any of *patterns*.
+  #
+  # The pattern syntax is similar to shell filename globbing, see `File.match?` for details.
+  #
+  # **options:**
+  # * *allow_dots*: Match hidden files if `true` (default: `false`).
+  #
+  # NOTE: Path separator in patterns needs to be always `/`. The returned file names use system-specific path separators.
+  def self.glob(*patterns, **options) : Array(String)
```

Comment:
Using `**options` is easier, but please duplicate the definitions so that the documentation looks nice.